ui/vite.config.ts

```diff
@@ -1,8 +1,17 @@
+/// <reference types="vitest" />
+/// <reference types="vite/client" />
+
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import viteTsconfigPaths from "vite-tsconfig-paths";
 
 // https://vitejs.dev/config/
 export default defineConfig({
 	plugins: [react(), viteTsconfigPaths()],
+	test: {
```

Comment:
Add the doc link

Reply:
https://github.com/vitest-dev/vitest/tree/main/examples/react-testing-lib